src/components/SetStateDialog.vue

```diff
@@ -17,7 +17,8 @@ export default {
   },
   methods: {
     onIntersect([entry]) {
-      this.$apollo.queries.taskRunIds.skip = !entry.isIntersecting
+      this.$apollo.queries.taskRunIds.skip =
+        !entry.isIntersecting || !this.flowRun
```

Comment:
Could the task run id query error out if this intersection method hasn't run? It seems like this would have to run for that query to ever meet skip requirements

Reply:
order switched